tests/unit/test_cube.py

```diff
@@ -75,6 +105,18 @@ def cube_response_type_fixture(self, request):
         cube_response, expected_value = request.param
         return cube_response, expected_value
 
+    @pytest.fixture(
+        params=[
+            ((DT.CAT, DT.MR, DT.MR), ("alias1", "alias2", "alias2"), True),
+            ((DT.CAT, DT.MR), ("alias1", "alias2"), False),
+            ((DT.CAT, DT.MR, DT.MR), ("alias1", "alias2", "alias3"), False),
+            ((DT.CAT, DT.TEXT, DT.TEXT), ("alias1", "alias2", "alias2"), False),
+        ]
+    )
+    def cube_dimensions_fixture(self, request):
+        dimension_types, aliases, expected_value = request.param
+        return dimension_types, aliases, expected_value
+
```

Comment:
Place fixtures in alphabetical order. This aids locating them when search is unavailable.